tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -71,6 +71,52 @@ def test_fsx_lustre(
     _test_data_repository_task(remote_command_executor, mount_dir, bucket_name, fsx_fs_id, region)
 
 
+@pytest.mark.parametrize("deployment_type, per_unit_storage_throughput", [("PERSISTENT_1", 200)])
+@pytest.mark.regions(["us-east-1"])
+@pytest.mark.instances(["c5.xlarge", "m6g.xlarge"])
+@pytest.mark.schedulers(["sge"])
+@pytest.mark.usefixtures("os", "instance", "scheduler", "deployment_type")
+# FSx is not supported on CentOS 6
+@pytest.mark.skip_oss(["centos6"])
+# FSx is only supported on ARM instances for Ubuntu 18.04 and Amazon Linux 2
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "alinux", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "centos7", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "ubuntu1604", "*")
+def test_fsx_lustre_backup(
+    deployment_type, per_unit_storage_throughput, region, pcluster_config_reader, clusters_factory, test_datadir, os,
+):
+    """
+    Test all FSx Lustre related features.
+
+    Grouped all tests in a single function so that cluster can be reused for all of them.
+    """
+    mount_dir = "/fsx_mount_dir"
+    cluster_config = pcluster_config_reader(
+        mount_dir=mount_dir, deployment_type=deployment_type, per_unit_storage_throughput=per_unit_storage_throughput,
+    )
```

Comment:
In the first cluster, can you set a `daily_automatic_backup_start_time` and then add a check that verifies a backup is created automatically, and that tags are copied to it?